### ISSUE TITLE
iccad 2015 contest benchmark has only simplified tech lef but need full

### ISSUE THREAD

**Issue by MyonForCode** [limbo018/DREAMPlace#186]
Hi Dr.Lin,

  Sorry to bother, but I was trying to carry on the physical design after global placement with DREAMPlace.
  When I set the init_lef_file in Innovus, an error came out saying the tech lef is not correct.
  ICCAD 2015 contest benchmarks is what I'm using, but the [tech.lef.gz](https://www.iccad-contest.org/2015/problem_C/Benchmarks/tech.lef.gz) cannot be found anymore.
  Is there any chance that you could provide the full tech.lef.gz?
  I would appreciate it if you can help me, thank you for all your effort. 

Matt

**Comment by limbo018** [comment]
Please check https://github.com/limbo018/DREAMPlace/blob/master/benchmarks/iccad2015.ot.md

This markdown document has the link for the benchmarks.

**Comment by MyonForCode** [comment]
Yes, I have downloaded it. But the benchmarks doesn't include the complete tech lef used for future PD / routing research.

![image](https://github.com/user-attachments/assets/efcb1885-3517-4b8c-b992-8ceadde37d72)

 The contest provided the link I mentioned earlier but cannot be found. So I would like to ask if you still have the complete tech lef provided at that time. 

 If you no longer have these files, please let me know. Thank you for your time on this issue.

**Comment by MyonForCode** [comment]
This is the format of the tech lef I need (you provided tech lef like this in ISPD 2015 benchmarks but with no libs). 
![image](https://github.com/user-attachments/assets/96835dde-5587-4e0f-b2a8-8af05fd4d63f)

This is the simplified version you provided in ICCAD 2015 benchmarks. 
![image](https://github.com/user-attachments/assets/7947cf82-e83b-40f4-a886-a417950f2612)

**Comment by limbo018** [comment]
We do not have the full tech file either. Maybe you can check with the contest chairs to see whether they have a copy.

**Comment by MyonForCode** [comment]
Thank you, Dr. Lin. I will contact with them. If I get the full version, I will post the link here. 

Thanks again.

**Comment by sezinata** [comment]
Hi @MyonForCode any updates on full version?  

Thank you
